wiki/Bazel-User-Guide.md

```diff
@@ -0,0 +1,277 @@
+## Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [Building the app](#building-the-app)
+- [Running Tests](#running-tests)
+  - [How to Obtain a Test Target](#how-to-obtain-a-test-target)
+  - [Common Flags](#common-flags)
+  - [Running multiple test targets](#running-multiple-test-targets)
+- [Concepts and Terminology](#concepts-and-terminology)
+- [Syncing the Project](#Syncing-the-project)
+
+## Overview
+Bazel is an open-source build and test tool similar to Make, Maven, and Gradle. It uses a human-readable, high-level build language.
+
+## Installation
+
+Follow the instructions on the [installation page](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android#install-bazel) to set up Bazel.
+
+### Building the app
+
+The [installation page](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android#install-bazel) provides information on running the project in Android Studio.
+
+You can also run the app from the command line using the commands detailed below.
+
+Using the command line is helpful in instances when you need to build and run some targets outside of Android Studio.
+
+Run the following commands in your terminal. All Bazel commands must be run from the root of the `oppia-android` directory otherwise they will fail.
+
+Note that on the first run, these commands may take 10-20 minutes to complete depending on the performance of your machine. Subsequent runs will be much faster.
+
+**On SDK 29 and below, run**:
+   ```
+   bazel mobile-install //:oppia_dev_binary
+   ```
+This will build, install and launch the app on your device.
+
+**On SDK 30 and newer, run**:
+   ```
+   bazel build //:oppia_dev
+   ```
+followed by:
+   ```
+   adb install bazel-bin/oppia_dev_binary.apk  
+   ```
+* Starting from Sdk 30, incremental builds, like those executed using `bazel mobile-install`, are no longer permitted, necessitating the use of two separate commands.
```

Comment:
Another case for `SDK`. Perhaps do a quick search across the wiki to ensure consistency?